jetbrains-core/src/software/aws/toolkits/jetbrains/services/caws/envclient/models/GetActivityResponse.kt

```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.caws.envclient.models
+
+data class GetActivityResponse(
+    val timestamp: String
```

Comment:
deserialization should automatically cast to Long?

Reply:
The API takes a string value so no cast required